be/src/storage/olap_define.h

```diff
@@ -48,8 +49,9 @@ static const size_t OLAP_PAGE_SIZE = 65536;
 
 static const uint64_t OLAP_FIX_HEADER_MAGIC_NUMBER = 0;
 
-// the max length supported for varchar type
-static const uint32_t OLAP_STRING_MAX_LENGTH = 1048576;
+uint32_t get_olap_string_max_length(){
```

Comment:
Can you move the implementation into the cpp files? Because olap_define is included by a lot of files.
If this implementation is moved into the cpp files, `#include "common/config.h"` can be moved into cpp files too.

Reply:
There is no existing cpp file. For better or worse, I just made the function inline. It also seems pretty common to make it an inline function in header if it's just a simple getter?

Reply:
The key point is not inline or not.
This relates to the compilation efficiency. If common/config.h is included in olap_define.h then all files including olap_define.h will include common/config.h, it will make the compilation a little slower.
If there is no corresponding .cpp file, you can create a new one and add it into be/src/storage/CMakeLists.txt.

Reply:
Let me know if you still prefer to have a cpp file, that's totally fine with me and I can make that change...

Reply:
Fair enough. PTAL.